packages/discovery-provider/src/api/v1/users.py

```diff
@@ -3169,3 +3170,33 @@ def get(self, receiving_user_id, grantor_user_id):
                 return success_response(None)
 
             return success_response(email_access.to_dict())
+
+
+# Comments
+user_comments_response = make_response(
+    "user_comments_response", ns, fields.List(fields.Nested(base_comment_model))
+)
+
+
+@ns.route("/<string:id>/comments")
+class UserComments(Resource):
```

Comment:
nit: Just to keep the SDK function names consistent, can we name this GetUserComments? (or, you can change the `id` field below, as that's what is used to generate the SDK function name)

Reply:
I kept the naming consistent with the tracks comments endpoint, do we want to change both to add get to them?

Reply:
Oh okay consistency good, let's leave it. Changing the existing ones would require a lot more coordination than it's worth since it's already in use.